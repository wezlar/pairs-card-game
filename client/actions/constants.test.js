import ACTION_TYPES from './constants';

describe(`actionCreators.js`, () => {
  describe(`ACTION_TYPES`, () => {
    test(`it matches the snapshot`, () => {
      expect(ACTION_TYPES).toMatchSnapshot();
    });
  });
});