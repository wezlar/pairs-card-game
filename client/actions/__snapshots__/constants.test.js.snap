// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actionCreators.js ACTION_TYPES it matches the snapshot 1`] = `
Object {
  "ADD_NEW_DECK": "ADD_NEW_DECK",
  "CARDS_MATCHED": "CARDS_MATCHED",
  "CARD_FLIP": "CARD_FLIP",
  "CARD_MATCHED": "CARD_MATCHED",
  "CARD_SELECTED": "CARD_SELECTED",
  "UPDATE_NUMBER_OF_CARDS": "UPDATE_NUMBER_OF_CARDS",
  "UPDATE_SCORE": "UPDATE_SCORE",
}
`;
